ports/esp32/boards/TINYPICO/mpconfigboard.mk

```diff
@@ -2,4 +2,5 @@ FLASH_FREQ = 80m
 
 SDKCONFIG += boards/sdkconfig.base
 SDKCONFIG += boards/sdkconfig.spiram
+SDKCONFIG += boards/sdkconfig.240mhz
```

Comment:
To be consistent, the 240mhz should come before spiram.

Reply:
Correct. Done.